Inventory/Model/Stock/Command/Get.php

```diff
@@ -27,6 +27,11 @@ class Get implements GetInterface
      */
     private $stockFactory;
 
+    /**
+     * @var StockInterface[]
+     */
+    private $stocks;
```

Comment:
Please, create cache object to wrap original logic, such approach provides ability to disable cache fo scenario where it's not needed. You may see example here: `\Magento\Inventory\Model\Source\Command\GetSourcesAssignedToStockOrderedByPriorityCache`

Reply:
code updated